docs/toree.md

```diff
@@ -0,0 +1,34 @@
+Get Started with XGBoost4J-Spark with Apache Toree Jupyter Notebook
+===================================================================
+This is a getting started guide to XGBoost4J-Spark using an [Apache Toree] (https://toree.apache.org/) Jupyter notebook. At the end of this guide, the reader will be able to run a sample notebook that runs on NVIDIA GPUs.
+
+Before you begin, please ensure that you have setup a [Spark Standalone Cluster](docs/standalone.md).
```

Comment:
Linked doc path is off, needs to be absolute or a relative path from this doc's directory

Reply:
done